sprint0/sprint0/BossesSpriteFactory.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace sprint0
+{
+    public class BossesSpriteFactory
+    {
+        Game1 game;
+
+        public BossesSpriteFactory(Game1 game)
+        {
+            this.game = game;
+        }
+
+        public ISprite MakeSprite(String spriteType, Vector2 location)
+        {
+            Texture2D texture;
+            texture = game.Content.Load<Texture2D>("Images/Bosses");
```

Comment:
can we move this to the constructor? (see Jesse's PlayerSpriteFactory)
it's still loading every time we call MakeSprite (my SPriteFactory is a bad example)